apps/studio/components/interfaces/Integrations/Webhooks/ListTab.tsx

```diff
@@ -38,6 +38,7 @@ export const WebhooksListTab = () => {
     <div className="p-10">
       <HooksList createHook={createHook} editHook={editHook} deleteHook={deleteHook} />
       <EditHookPanel
+        key={selectedHook?.id}
```

Comment:
whenever this changes, the internal state of `EditHookPanel` will be re-initialized